dips/dip-158.md

```diff
@@ -0,0 +1,794 @@
+|3|NOTIFY_AUTHORIZED|AuthorizedCommand|Sender|The wallet indicates the funds are locked for a future capture|
+| 
+
+
+### Capture Sequence (Auth / Capture)
+
+IMPORTANT: Successful authorization is a pre-condition to capture
+
+| Step |Functionality|Command Type|Triggered by | Description |
+|-|-|-|-|-|
+|1|CAPTURE_PAYMENT|CaptureCommand|Receiver|The receiver requests to capture the authorized funds|
+|2|NOTIFY_READY|ReadyCommand|Sender|The wallet indicates the payment is ready for settlement. The wallet is expected to submit a transaction on-chain immediately before or after this step|
+|
+
+### Abort Sequence - Customer Rejected the Payment Request
+
+| Step | Functionality |Command Type| Triggered by | Description |
+|-|-|-|-|-|
+|1|GET_INFO|InfoCommand|Sender|The wallet requests payment details using the reference id provided (i.e. the common payment identifier). The receiver will respond with the payment details and merchant data. The wallet will display the payment details so the customer can approve or reject|
+|2|ABORT_PAYMENT|AbortCommand|Sender|Customer rejected the payment request|
+| 
+
+### Void Sequence (Auth / Capture)
+
+IMPORTANT: Successful authorization is a pre-condition to void
+
+| Step |Functionality|Command Type|Triggered by | Description |
+|-|-|-|-|-|
+|1|VOID_PAYMENT|VoidCommand|Receiver|Receiving VASP wants to void the payment. For example, the merchant voided the payment because the purchased products are out of stock|
+|
+
+### Abort Sequence - Risk Checks Failed
+
+| Step | Functionality |Command Type| Triggered by | Description |
+|-|-|-|-|-|
+|1|GET_INFO|InfoCommand|Sender|The wallet requests payment details using the reference id provided (i.e. the common payment identifier). The receiver will respond with the payment details and merchant data. The wallet will display the payment details so the customer can approve or reject|
+|2|INIT_PAYMENT|InitCommand|Sender|Following customer approval the wallet initializes the payment process and sends the payer data.  Risk checks failed. Receiver will respond with abort status|
+| 
+
+
+### Abort Sequence - Merchant Data Checks (performed by the Sender) Failed
+
+| Step | Functionality |Command Type| Triggered by | Description |
+|-|-|-|-|-|
+|1|GET_INFO|InfoCommand|Sender|The wallet requests payment details using the reference id provided (i.e. the common payment identifier). The receiver will respond with the payment details and merchant data. The wallet will display the payment details so the customer can approve or reject|
+|2|INIT_PAYMENT|InitCommand|Sender|Following customer approval the wallet initializes the payment process and sends the payer data. The receiver will respond with appropriate success status|
+|3|ABORT_PAYMENT|AbortCommand|Sender|Sender decided to abort. Can happen if the sender does not approve merchant data provided by the receiver|
+|
+
+
+
+
+
+---
+# Appendix A - Pre-requisite: Sharing Common Identifier and Address
```

Comment:
should this be "payment identifier" ?

Reply:
In the context of this DIP, yes, it is a payment identifier. I will update.
In a broader perspective, the identifier can be that of fund pre-approval request.